search/search_repository.go

```diff
@@ -317,6 +340,21 @@ func parseArray(anArray []interface{}, l *[]Query) {
 	}
 }
 
+// ResultViewType represents the view type requested for the result
+type ResultViewType int
+
+// List of available view types requested for the result
+const (
+	ResultViewTree ResultViewType = iota
+	ResultViewList
+)
+
+// QueryOptions represents all options provided user
+type QueryOptions struct {
+	ResultView   ResultViewType
```

Comment:
Instead of `ResultView ResultViewType`. Wouldn't it be better to have a simple `TreeView bool`?  
1 - tree view
0 - no tree view i.e. list view

Reply:
Yeah. That looks better. I hope there won't be another type.

Reply:
Fixed in 90156bf268dee2adde5da7738f3cd0de5baf9ad8